frontend/src/app/messages/messages.component.ts

```diff
@@ -0,0 +1,38 @@
+import { Component } from '@angular/core';
+import { Http } from '@angular/http';
```

Comment:
Please migrate to new HttpClient

Reply:
from @angular/common/http

Reply:
good catch! I will update it to use new http infrastructure!